### PR TITLE
Update installation documents.

### PR DESCRIPTION
**Pull request by masteryyh**
Updated new installation mode and known issues & workarounds, deleted/updated outdated pictures.

**Related issue**
https://github.com/harvester/docs/issues/373

### CODE REVIEW

File: docs/install/iso-install.md [hunk 1]
```diff
@@ -28,30 +28,60 @@ Note: This [video](https://youtu.be/X0VIGZ_lExQ) shows a quick overview of the I
...
+
+1. Choose installation mode: 
+   - Create a new Harvester cluster;
       - Note: By default, the first node will be the management node of the cluster. When there are 3 nodes, the other 2 nodes added first are automatically promoted to management nodes to form an HA cluster.
-      
+
       If you want to promote management nodes from different zones, you can add the node label `topology.kubernetes.io/zone` in the [os.labels](./harvester-configuration.md#oslabels) config by providing a URL of [Harvester configuration](./harvester-configuration.md) on the customize the host step. In this case, at least three different zones are required.
-1. Choose the installation device on which the Harvester cluster will be installed
+   - Join an existing Harvester cluster;
+   - Install Harvester binaries only;
```
Comment (vickyhella) [inline docs/install/iso-install.md:38]:
```suggestion
   - Join an existing Harvester cluster.
   - Install Harvester binaries only.
```

File: docs/install/iso-install.md [hunk 2]
```diff
@@ -28,30 +28,60 @@ Note: This [video](https://youtu.be/X0VIGZ_lExQ) shows a quick overview of the I
...
+      - Note: When choosing `Install Harvester binaries only`, additional setup is required after first boot up.
```
Comment (vickyhella) [inline docs/install/iso-install.md:39]:
```suggestion
      - Note: If you choose `Install Harvester binaries only`, additional setup is required after the first bootup.
```

File: docs/install/iso-install.md [hunk 3]
```diff
@@ -28,30 +28,60 @@ Note: This [video](https://youtu.be/X0VIGZ_lExQ) shows a quick overview of the I
...
-   ![iso-install-disk.png](/img/v1.2/install/iso-install-disk.png)
-1. We recommend choosing a separate disk to store VM data.
-   ![iso-install-disk.png](/img/v1.2/install/iso-select-data-disk.png )
-1. Configure the hostname and select the network interface for the management network. By default, Harvester will create a bonded NIC named `mgmt-bo`, and the IP address can be configured via DHCP or a statically assigned one <small>(Note: The Node IP can not change at the lifecycle of a Harvester cluster. If DHCP is used, users must make sure the DHCP server always offers the same IP for the same Node. If Node IP is changed, the related Node cannot join the cluster and might even break the cluster)</small>.
 
-   ![iso-installed.png](/img/v1.2/install/iso-nic-config.png)
+      When you have only one disk, or using the same disk for both OS and VM data, you will need to configure persistent partition size, which is used to store system packages and container images, the default and minimum value is 150 GB.
```
Comment (vickyhella) [inline docs/install/iso-install.md:45]:
```suggestion
      If you only have one disk, or have to use the same disk for both OS and VM data, you need to configure persistent partition size, which is used to store system packages and container images (the default and minimum value is 150 GB).
```

File: docs/install/iso-install.md [hunk 4]
```diff
@@ -28,30 +28,60 @@ Note: This [video](https://youtu.be/X0VIGZ_lExQ) shows a quick overview of the I
...
+   :::info
+
+   Choosing a separate disk to store VM data is recommended.
+
+   :::
+
+1. Configure the hostname of this node.
+   ![config-hostname.png](/img/v1.2/install/config-hostname.png)
+
+1. Configure network interface(s) for the management network. By default, Harvester will create a bonded NIC named `mgmt-bo`, and the IP address can be configured via DHCP or a statically assigned one.
```
Comment (vickyhella) [inline docs/install/iso-install.md:56]:
```suggestion
1. Configure network interface(s) for the management network. By default, Harvester will create a bonded NIC named `mgmt-bo`, and the IP address can be configured via DHCP or statically assigned.
```

File: docs/install/iso-install.md [hunk 5]
```diff
@@ -28,30 +28,60 @@ Note: This [video](https://youtu.be/X0VIGZ_lExQ) shows a quick overview of the I
...
+   ![config-network.png](/img/v1.2/install/config-network.png)
+      - Note: The Node IP can not change at the lifecycle of a Harvester cluster. If DHCP is used, users must make sure the DHCP server always offers the same IP for the same Node. If Node IP is changed, the related Node cannot join the cluster and might even break the cluster.
```
Comment (vickyhella) [inline docs/install/iso-install.md:58]:
```suggestion
      - Note: The node IP cannot be changed throughout the lifecycle of a Harvester cluster. If DHCP is used, you must make sure the DHCP server always offers the same IP for the same node. If the node IP is changed, the related node cannot join the cluster and might even break the cluster.
```

File: docs/install/iso-install.md [hunk 6]
```diff
@@ -28,30 +28,60 @@ Note: This [video](https://youtu.be/X0VIGZ_lExQ) shows a quick overview of the I
...
+
 1. (Optional) Configure the DNS servers. Use commas as a delimiter.
-1. Configure the `Virtual IP` which you can use to access the cluster or join other nodes to the cluster <small>(Note: If your IP address is configured via DHCP, you will need to configure static MAC-to-IP address mapping on your DHCP server in order to have a persistent Virtual IP, VIP must be different than any Node IP)</small>.
+
+1. Configure the `Virtual IP` which you can use to access the cluster or join other nodes to the cluster.
```
Comment (vickyhella) [inline docs/install/iso-install.md:62]:
```suggestion
1. Configure the `Virtual IP` which you can use to access the cluster or add other nodes to the cluster.
```

File: docs/install/iso-install.md [hunk 7]
```diff
@@ -28,30 +28,60 @@ Note: This [video](https://youtu.be/X0VIGZ_lExQ) shows a quick overview of the I
...
+   - Note: If your IP address is configured via DHCP, you will need to configure static MAC-to-IP address mapping on your DHCP server in order to have a persistent Virtual IP, VIP must be different than any Node IP.
```
Comment (vickyhella) [inline docs/install/iso-install.md:63]:
```suggestion
   - Note: If your IP address is configured via DHCP, you need to configure a static MAC-to-IP address mapping on your DHCP server in order to have a persistent virtual IP (VIP), and the VIP must be unique.
```

File: docs/install/iso-install.md [hunk 8]
```diff
@@ -28,30 +28,60 @@ Note: This [video](https://youtu.be/X0VIGZ_lExQ) shows a quick overview of the I
...
+
 1. (Optional) You can choose to import SSH keys from a remote server URL. Your GitHub public keys can be used with `https://github.com/<username>.keys`.
+
 1. (Optional) If you need to customize the host with a [Harvester configuration](./harvester-configuration.md) file, enter the HTTP URL here.
+
 1. After confirming the installation options, Harvester will be installed to your host. The installation may take a few minutes to be complete.
-1. Once the installation is complete, it will restart the host. After the restart, the Harvester console containing the management URL and status will be displayed. You can use `F12` to switch from the Harvester console to the Shell and type `exit` to go back to the Harvester console.
+
+1. Once the installation is complete, your node will be restarted. After the restart, the Harvester console containing the management URL and status will be displayed. You can use `F12` to switch from the Harvester console to the Shell and type `exit` to go back to the Harvester console.
+   - Note: If you choose `Install Harvester binaries only` at the first page, additional setup is required after first boot up.
```
Comment (vickyhella) [inline docs/install/iso-install.md:80]:
```suggestion
   - Note: If you choose `Install Harvester binaries only` on the first page, additional setup is required after the first bootup.
```

File: docs/install/iso-install.md [hunk 9]
```diff
@@ -68,9 +98,16 @@ In some cases, if you are using an older graphics card/monitor, you may encounte
...
+
+If you are using a version earlier than v1.1.1, please try the following workaround:
+
 1. Boot up with the ISO, and press `E` to edit the first menu entry:
-   ![grub-menu.png](/img/v1.2/install/grub-menu.png)
+   ![grub-menu.png](/img/v1.1/install/grub-menu.png)
+
 2. Append `vga=792` to the line started with `$linux`:
-   ![edit-menu-entry.png](/img/v1.2/install/edit-menu-entry.png)
+   ![edit-menu-entry.png](/img/v1.1/install/edit-menu-entry.png)
```
Comment (vickyhella) [inline docs/install/iso-install.md:111]: This is a v1.2 doc and it should reference images from `/img/v1.2`. Please double-check if the current images are fine, or should we copy the img to `/img/v1.2` and reference them from there instead

File: docs/install/usb-install.md [hunk 10]
```diff
@@ -11,12 +11,22 @@ title: "USB Installation"
 ## Create a bootable USB flash drive
 
 There are a couple of ways to create a USB installation flash drive.
+:::caution
+
+Your USB device will be erased after creating bootable device, no matter which tool you use. Please make sure all data on your USB device is backed up before creating bootable device.
```
Comment (vickyhella) [inline docs/install/usb-install.md:16]:
```suggestion
Your USB device will be erased after creating a bootable device, no matter which tool you use. Please make sure all data on your USB device is backed up before creating a bootable device.
```

File: docs/install/usb-install.md [hunk 11]
```diff
@@ -11,12 +11,22 @@ title: "USB Installation"
...
+
+:::
+
+### rufus
 
+[rufus](https://rufus.ie/) supports creating bootable devices using bootable images under Windows.
```
Comment (vickyhella) [inline docs/install/usb-install.md:22]:
```suggestion
[rufus](https://rufus.ie/) supports creating bootable devices using bootable images in Windows.
```

File: docs/install/usb-install.md [hunk 12]
```diff
@@ -11,12 +11,22 @@ title: "USB Installation"
...
+![rufus.png](/img/v1.2/install/rufus.png)
+  - Note: Both `DD mode` and `ISO mode` works. `DD mode` works just like `dd` command under Linux, and you can't browse partitions after you created bootable device. `ISO mode` creates partitions on your device automatically and copies files to these partitions and you can browse these partitions even after you created bootable device.
```
Comment (vickyhella) [inline docs/install/usb-install.md:24]:
```suggestion
  - Note: Both `DD mode` and `ISO mode` works. `DD mode` works just like the `dd` command in Linux, and you can't browse partitions after you created a bootable device. `ISO mode` creates partitions on your device automatically and copies files to these partitions and you can browse these partitions even after you created a bootable device.
```

File: i18n/zh/docusaurus-plugin-content-docs/current/install/iso-install.md [hunk 13]
```diff
@@ -22,32 +22,62 @@ Description: 如果需要获取 Harvester ISO，访问 GitHub 上的发行版本
...
 
-1. 安装 Harvester ISO 磁盘并通过选择 `Harvester Installer` 来启动服务器。
+1. 挂载 Harvester ISO 磁盘并通过选择 `Harvester Installer` 来启动服务器。
    ![iso-install.png](/img/v1.2/install/iso-install.png)
-1. 通过创建新的 Harvester 集群或加入现有集群来选择安装模式。
-   - 注意：默认情况下，第一个节点将是集群的管理节点。当有 3 个节点时，首先添加的另外 2 个节点会自动提升为管理节点，从而形成 HA 集群。
 
-   如果你想提升其它地区的管理节点，你可以在执行自定义主机的步骤时提供一个 [Harvester 配置](./harvester-configuration.md)的 URL，在 [os.labels](./harvester-configuration.md#oslabels) 中添加节点标签 `topology.kubernetes.io/zone`。在这种情况下，至少需要三个不同的地区。
+1. 选择一个安装模式：
+   - 创建一个新的Harvester集群；
```
Comment (vickyhella) [inline i18n/zh/docusaurus-plugin-content-docs/current/install/iso-install.md:29]:
```suggestion
   - 创建一个新的 Harvester 集群；
```

File: i18n/zh/docusaurus-plugin-content-docs/current/install/iso-install.md [hunk 14]
```diff
@@ -22,32 +22,62 @@ Description: 如果需要获取 Harvester ISO，访问 GitHub 上的发行版本
...
+      - 注意：默认情况下，第一个节点将是集群的管理节点。当有 3 个节点时，首先添加的另外 2 个节点会自动提升为管理节点，从而形成 HA 集群。
+
+      如果你想提升其它地区的管理节点，你可以在执行自定义主机的步骤时提供一个 [Harvester 配置](./harvester-configuration.md)的 URL，在 [os.labels](./harvester-configuration.md#oslabels) 中添加节点标签 `topology.kubernetes.io/zone`。在这种情况下，至少需要三个不同的地区。
+   - 加入一个现有的Harvester集群；
```
Comment (vickyhella) [inline i18n/zh/docusaurus-plugin-content-docs/current/install/iso-install.md:33]:
```suggestion
   - 加入一个现有的 Harvester 集群。
```

File: docs/install/iso-install.md [hunk 15]
```diff
@@ -28,30 +28,60 @@ Note: This [video](https://youtu.be/X0VIGZ_lExQ) shows a quick overview of the I
 
 1. Mount the Harvester ISO disk and boot the server by selecting the `Harvester Installer` option.
    ![iso-install.png](/img/v1.2/install/iso-install.png)
-1. Choose the installation mode by either creating a new Harvester cluster, or by joining an existing one.
+
+1. Choose installation mode: 
+   - Create a new Harvester cluster;
```
Comment (vickyhella) [inline docs/install/iso-install.md:33]:
```suggestion
   - Create a new Harvester cluster.
```

File: i18n/zh/docusaurus-plugin-content-docs/current/install/iso-install.md [hunk 16]
```diff
@@ -22,32 +22,62 @@ Description: 如果需要获取 Harvester ISO，访问 GitHub 上的发行版本
...
+   - 仅安装Harvester系统文件；
```
Comment (vickyhella) [inline i18n/zh/docusaurus-plugin-content-docs/current/install/iso-install.md:34]:
```suggestion
   - 仅安装 Harvester 系统文件。
```

File: i18n/zh/docusaurus-plugin-content-docs/current/install/requirements.md [hunk 17]
```diff
@@ -15,11 +15,11 @@ Harvester 是运行在裸机服务器上的 HCI 解决方案，要正常运行
 
 | 类型 | 要求和推荐 |
 |:-----------------|:------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| CPU | 仅支持 x86_64。需要硬件辅助虚拟化。8 核（至少）用于测试；生产中推荐使用 16 核或以上 |
-| 内存 | 32 GB（至少）用于测试；生产中推荐使用 64 GB 或以上 |
-| 磁盘容量 | 250 GB（至少）用于测试；生产中推荐使用 500 GB 或以上 |
+| CPU | 仅支持 x86_64。需要硬件辅助虚拟化。8 核（至少）用于测试；生产中需要使用 16 核或以上 |
+| 内存 | 32 GB（至少）用于测试；生产中需要使用 64 GB 或以上 |
+| 磁盘容量 | 250 GB（至少）用于测试（使用多盘布局时至少180 GB）；生产中需要使用 500 GB 或以上 |
```
Comment (vickyhella) [inline i18n/zh/docusaurus-plugin-content-docs/current/install/requirements.md:20]:
```suggestion
| 磁盘容量 | 250 GB（至少）用于测试（使用多盘布局时至少 180 GB）；生产中需要使用 500 GB 或以上 |
```

File: i18n/zh/docusaurus-plugin-content-docs/current/install/usb-install.md [hunk 18]
```diff
@@ -7,11 +7,22 @@ title: "U 盘安装"
 ## 创建一个 USB 启动盘
 
 你可以通过以下几种方式创建 USB 安装启动盘。
+:::caution
+
+在创建可引导设备后，你的 USB 设备上的所有数据将会被清除，无论你使用什么工具。请保证在创建可引导设备前备份了你的USB设备上所有的数据。
```
Comment (vickyhella) [inline i18n/zh/docusaurus-plugin-content-docs/current/install/usb-install.md:12]:
```suggestion
在创建可引导设备后，你的 USB 设备上的所有数据将会被清除，无论你使用什么工具。请保证在创建可引导设备前备份了你的 USB 设备上所有的数据。
```

**Review by vickyhella — COMMENTED** [review]
Only some nits!

File: docs/install/management-address.md [hunk 19]
```diff
@@ -37,5 +37,5 @@ Example of output:
 The management address has two usages.
```
Comment (vickyhella) [inline docs/install/management-address.md:37]:
```suggestion
The management address:
```

File: docs/install/iso-install.md (hunk 9 above)
Comment (masteryyh) [reply docs/install/iso-install.md:111]: It's a typo LOL 🤣

File: i18n/zh/docusaurus-plugin-content-docs/current/install/usb-install.md [hunk 20]
```diff
@@ -7,11 +7,27 @@ title: "U 盘安装"
...
+在创建可引导设备后，你的 USB 设备上的所有数据将会被清除，无论你使用什么工具。请保证在创建可引导设备前备份了你的 USB 设备上所有的数据。
+
+:::
+
+### rufus
 
+[rufus](https://rufus.ie/)支持在 Windows 系统下使用可引导镜像来创建可引导设备。
+![rufus.png](/img/v1.2/install/rufus.png)
+
+:::note
```
Comment (vickyhella) [inline i18n/zh/docusaurus-plugin-content-docs/current/install/usb-install.md:21]:
```suggestion
:::info
```

File: docs/install/iso-install.md [hunk 21]
```diff
@@ -28,30 +28,62 @@ Note: This [video](https://youtu.be/X0VIGZ_lExQ) shows a quick overview of the I
...
-   ![iso-install-disk.png](/img/v1.2/install/iso-install-disk.png)
-1. We recommend choosing a separate disk to store VM data.
-   ![iso-install-disk.png](/img/v1.2/install/iso-select-data-disk.png )
-1. Configure the hostname and select the network interface for the management network. By default, Harvester will create a bonded NIC named `mgmt-bo`, and the IP address can be configured via DHCP or a statically assigned one <small>(Note: The Node IP can not change at the lifecycle of a Harvester cluster. If DHCP is used, users must make sure the DHCP server always offers the same IP for the same Node. If Node IP is changed, the related Node cannot join the cluster and might even break the cluster)</small>.
 
-   ![iso-installed.png](/img/v1.2/install/iso-nic-config.png)
+      If you only have one disk, or have to use the same disk for both OS and VM data, you need to configure persistent partition size, which is used to store system packages and container images (the default and minimum value is 150 GB).
```
Comment (guangbochen) [inline docs/install/iso-install.md:47]:
```suggestion
      If you only have one disk, or have to use the same disk for both OS and VM data, you need to configure persistent partition size, which is used to store system packages and container images (the default and minimum value is 150 GiB).
```

**Review by guangbochen — APPROVED** [review]
LGTM, thanks.
Note: will need to replace the rc5-image after the official release.

File: docs/install/iso-install.md (hunk 21 above)
Comment (masteryyh) [reply docs/install/iso-install.md:47]: Do we need to change all `GB` to `Gi` or `GiB`?

File: docs/install/usb-install.md [hunk 22]
```diff
@@ -11,12 +11,27 @@ title: "USB Installation"
 ## Create a bootable USB flash drive
 
 There are a couple of ways to create a USB installation flash drive.
+:::caution
+
+Your USB device will be erased after creating a bootable device, no matter which tool you use. Please make sure all data on your USB device is backed up before creating a bootable device.
```
Comment (LucasSaintarbor) [inline docs/install/usb-install.md:16]:
```suggestion
No matter which tool you use, your USB device data is erased after creating a bootable device. Please back up all data on your USB device before creating a bootable device.
```

File: docs/install/usb-install.md [hunk 23]
```diff
@@ -11,12 +11,27 @@ title: "USB Installation"
...
+:::
+
+### rufus
 
+[rufus](https://rufus.ie/) supports creating bootable devices using bootable images in Windows.
+![rufus.png](/img/v1.2/install/rufus.png)
+
+:::info
+
+Both `DD mode` and `ISO mode` works. `DD mode` works just like the `dd` command in Linux, and you can't browse partitions after you created a bootable device. `ISO mode` creates partitions on your device automatically and copies files to these partitions and you can browse these partitions even after you created a bootable device.
```
Comment (LucasSaintarbor) [inline docs/install/usb-install.md:27]:
```suggestion
Both `DD mode` and `ISO mode` works. `DD mode` works just like the `dd` command in Linux, and you can't browse partitions after you create a bootable device. `ISO mode` creates partitions on your device automatically and copies files to these partitions, and you can browse these partitions even after you create a bootable device.
```

**Review by LucasSaintarbor — APPROVED** [review]
LGTM 👍 Thanks @masteryyh! Merging now

**Pull request merged**